changelogs/fragments/601-copy-loadlib-member.yml

```diff
@@ -0,0 +1,3 @@
+bugfixes:
+- zos_copy - Copy failed from a loadlib member to another loadlib member. Fix now looks for error in stdout in the if statement to use -X option.
```

Comment:
I think the fragment could be reworded to something like:
Fixes a bug in a conditional statement that caused a failure when copying a loadlib member to another one.

Reply:
I had a similar wording originally, but went this route to follow ansible development guidelines for fragments: (https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment) "Use past tense to describe the problem and present tense to describe the fix."

Reply:
I did not even know they had a recommended guideline for this, learned something new.